src/parser/tests/parser.test.js

```diff
@@ -0,0 +1,27 @@
+const assert = require('assert');
+const expect = require('chai').expect;
+const path = require('path');
+const expectedOutputs = require('./expectedOutputs');
+const {parseFiles} = require('../parseFiles');
+describe('Parser Tests',function() {
+  let asyncApi;
+  it('1# Should be able to parse correctly formatted asyncApi and scenario files.',async function() {
```

Comment:
You dont need the `1#` for the tests 🙂 Same for `2#`

Reply:
i think that could be usefull tho if sm1 want to refer to a test.

Reply:
I have never had the use-case 😄 Usually when you refer to talk about which tests are added you read it as `Parser Tests Should be able to parse ....`. It ruins the flow if `#1` is present 🙂

Reply:
ok